src/cityreader.py

```diff
@@ -15,26 +22,41 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
-
 cities = []
+with open('src/cities.csv', newline='') as csvfile:
```

Comment:
When I tried to run this it said this file did not exist. I think it's just a path issue. Since they're in the same directory I don't think you need the src. It may work on your setup depending on your file structure.